### ISSUE TITLE
[Question] Other GelBooru based boorus

### ISSUE THREAD

**Issue by Donny-Pandairaz** [Enchoseon/gelbooru-overhaul-userscript#3]
Just wanted to ask a question. Would it be possible to adjust this userscript to work with other Gelbooru-based boorus like rule34.xxx given that Gelbooru functions on Beta 0.2.5 and the later on Beta 0.2? I tried messing around with it a bit and I can't seem to get it to display the display windows or sidebar due to rule34.xxx lacking a secondary  "body" that Gelbooru does with the "container" ID

**Comment by Enchoseon** [comment]
Support for other boorus has not been attempted or planned for [maintainability reasons](https://github.com/Enchoseon/gelbooru-overhaul-userscript/pull/1#issuecomment-1236195668). This userscript does lots of hacks and tweaks specific to Gelbooru.com; not necessarily Gelbooru the booru software. It's not built for that.

A tool with proper multi-booru support would use a completely different approach than this userscript; that of actually using booru APIs rather than their individual HTML frontends.

If you're looking for universal multi-booru poweruser features and theming, [Grabber](https://github.com/Bionus/imgbrd-grabber) and [LoliSnatcher](https://github.com/NO-ob/LoliSnatcher_Droid) may serve you better.

*tl;dr It's possible, but definitely not something I'm gonna try or recommend to anyone who isn't a masochist lol.*

**Comment by Donny-Pandairaz** [comment]
I see, I see. Makes sense. I was looking for something that has a blacklisting feature like this script and Danbooru Misc Tweaks, but for Rule34.xxx but it seems that no one has ever done it.
_Guess that means it's time to become a masochist lol_
